content/en/coterm/usage.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Using Datadog CoTerm
+further_reading:
+- link: "/coterm"
+  tag: "documentation"
+  text: "Datadog CoTerm"
+- link: "/coterm/install"
+  tag: "documentation"
+  text: "Install Datadog CoTerm"
+- link: "/coterm/rules"
+  tag: "documentation"
+  text: "CoTerm Configuration Rules"
+---
+
+## CoTerm CLI command structure
+
+```shell
+ddcoterm [OPTIONS] [-- <COMMAND>...] [COMMAND]
+```
+
+Run `ddcoterm --help` for all options and commands.
+
+## Record a terminal session
+
+CoTerm records terminal sessions that you can play back and review in Datadog. For your security, sensitive data (such as passwords and API keys) are [automatically redacted][1]. Any processes launched in the terminal session are recorded as [events][2].
+
+### Launch a recorded terminal session
+To manually launch Datadog CoTerm and record the entirety of your terminal session:
+
+```shell
+ddcoterm
+```
+
+When you end the session, CoTerm stops recording and sends the captured process data to Datadog.
+
+### Record the output of a command
+To run an individual command and record its output:
+
+```shell
+ddcoterm -- datadog-agent status
+```
+
+This launches CoTerm and runs `datadog-agent status`. When the process completes, CoTerm stops recording and sends the captured process data to Datadog.
+
+## Automatically record a command
+
+To configure CoTerm to automatically record a particular command, create a shim:
+
+```shell
+ddcoterm shim create datadog-agent
+```
+
+After you create a shim, restart your terminal or source your profile. (For example, run `source ~/.bashrc`.) If you are using a shell other than Bash or Zsh, add `path/to/.ddcoterm/overrides` to your PATH manually.
+
+All future invocations of `datadog-agent` are automatically recorded.
+
+## Protect against dangerous terminal commands
+
+To prevent the accidental execution of designated terminal commands, you can configure CoTerm to act as a linter. For more control, you can use CoTerm with [Datadog Case Management][3] to require approval for designated commands.
+
+### Lint a command
+
+When you try to execute a designated command (for example, `kubectl scale`), CoTerm can display warnings and prompt you for confirmation.
+
+1. Create a shim for your command: `ddcoterm shim create kubectl`
+
+1. Configure a linting rule in your `.ddcoterm/config.yaml` file. For details on how to configure linting in CoTerm, see [CoTerm Configuration Rules][4].
+
+{{< code-block lang="yaml" filename=".ddcoterm/config.yaml" disable_copy="true" collapsible="true" >}}
+process_config:
+  commands:
+    - command: "kubectl"
+      lints:
+        - |
+          if has_arg("scale") and flags.context == nil then
+            return string.format("No kubectl context specified (effective context: '%s'). It is recommended to always explicitly specify the context when running `kubectl scale`.", k8s_context)
+          end
+{{< /code-block >}}
```

Comment:
To indent the code block so it's aligned with the rest of the content in step 2:
```suggestion
   {{< code-block lang="yaml" filename=".ddcoterm/config.yaml" disable_copy="true" collapsible="true" >}}
process_config:
  commands:
    - command: "kubectl"
      lints:
        - |
          if has_arg("scale") and flags.context == nil then
            return string.format("No kubectl context specified (effective context: '%s'). It is recommended to always explicitly specify the context when running `kubectl scale`.", k8s_context)
          end
   {{< /code-block >}}
```